.vscode/settings.json

```diff
@@ -23,5 +23,6 @@
     },
     "python.linting.pylintEnabled": false,
     "diffEditor.ignoreTrimWhitespace": false,
-    "python.pythonPath": "${workspaceFolder}/dev-env/bin/python"
+    "python.pythonPath": "/Users/anupkalburgi/projects/daml/dev-env/bin/python",
```

Comment:
That doesn’t seem like something we should commit.

Reply:
Yup, done how that ended up there.